packages/instant/src/components/ui/button.tsx

```diff
@@ -3,6 +3,11 @@ import * as React from 'react';
 
 import { ColorOption, styled } from '../../style/theme';
 
+export enum ButtonHoverStyle {
```

Comment:
Is this only to support the close `Icon`? I would bake click functionality directly into `Icon`. This type of configuration for the button seems like too much, all buttons should just behave the same way I think...